src/main/java/uk/gov/di/ipv/cri/common/library/persistence/item/SessionItem.java

```diff
@@ -25,6 +25,7 @@ public class SessionItem {
     private String persistentSessionId;
     private String clientSessionId;
     private String clientIpAddress;
+    private int dlAttemptCount;
```

Comment:
I think it would be worthwhile renaming this field  (& it's accessor/mutator methods) to something more generic such as `attemptCount`. Then it could be referenced by any CRI without any ambiguity about the meaning of `dl`.

If this field is only going to be updated after a sessionItem instance has been persisted then no other changes are required. However, if it desired for the attemptCount to be set to 1 (for example) when the sessionItem is first persisted then the SessionService class will most likely need an update